bundles/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/internal/handler/PigpioDigitalOutputHandler.java

```diff
@@ -63,6 +63,10 @@ public PigpioDigitalOutputHandler(GPIOOutputConfiguration configuration, JPigpio
             throw new NoGpioIdException();
         }
         this.gpio = new GPIO(jPigpio, gpioId, JPigpio.PI_OUTPUT);
+        if (configuration.invert) {
+            updateStatus.accept(OnOffType.ON);
```

Comment:
Should this line be added ? Because inverted means that pin should be by default in state 1, but this means OFF not ON, because it is inverted .

Reply:
Good point, I'm not sure on this on either. Maybe not setting `updateStatus` would be the correct thing to do. What do you think @nils-bauer ?

Reply:
AS 6 months have passsed, is this properly addressed ?

Reply:
I was hoping that this entire PR could be replaced with #13643 and still keep my fingers crossed that it will be merged over the next days.